server/src/pages/shot/view.js

```diff
@@ -298,12 +298,13 @@ class Body extends React.Component {
     }
 
     let clipUrl = null;
+    let clipDownloadUrl = null;
     if (clipNames.length) {
       let clipId = clipNames[0];
       let clip = this.props.shot.getClip(clipId);
       clipUrl = clip.image.url;
+      clipDownloadUrl = clipUrl + `?download=${encodeURIComponent(this.props.shot.filename)}`;
```

Comment:
This value needs to be signed by the server, so someone can't do something like: `http://cdnlink/image/uuid?download=somethingevil.exe`